project/project-info.conf

```diff
@@ -78,7 +78,7 @@ project-info {
     levels: [
       {
         readiness: Supported
-        since: "2019-12-31"
+        since: "2019-11-05"
         since-version: "2.6.0"
       }
       {
```

Comment:
For below Incubating 2.5.2 2017-05-24 would be more correct. Yeah, it has been many iterations, and slow progress during long periods.

https://akka.io/blog/news/2017/05/24/akka-2.5.2-released

That was when the development branch was merged to master and included in the ordinary 2.5.x patch releases.